### PR TITLE
Skip --env tests because they stall on CI

### PR DESCRIPTION
**Pull request by ManasJayanth**

### CODE REVIEW

File: .ci/esy-build-steps.yml [hunk 1]
```diff
@@ -12,21 +13,24 @@ steps:
...
     displayName: "npm run test: after unpack (iteration 1)"
-  - script: node postinstall.js
-    displayName: "node postinstall.js (iteration 2)"
-  - script: npm run test
-    displayName: "npm run test: after unpack (iteration 2)"
-  - script: node postinstall.js
-    displayName: "node postinstall.js (iteration 3)"
-  - script: npm run test
-    displayName: "npm run test: after unpack (iteration 3)"
+  # - script: node postinstall.js
```
Comment (ulrikstrid) [inline .ci/esy-build-steps.yml:28]: Is this part correct?

Comment (callistonianembrace) [reply .ci/esy-build-steps.yml:28]: I have no idea why we iterate so many times. @bryphe Do you happen to remember why?

Comment (ManasJayanth) [reply .ci/esy-build-steps.yml:28]: I have no idea why we iterate so many times. @bryphe Do you happen to remember why?

Comment (ulrikstrid) [reply .ci/esy-build-steps.yml:28]: Was just unsure if it was meant to be here or something that was forgotten. This is fine by me :)

**Pull request merged**